lib/assertions/whitelisting.js

```diff
@@ -171,13 +171,31 @@ const assertWhitelistAddressPresence = async (rskTxHelper, btcAddress, present)
   
 };
 
+/**
+ * 
+ * @param {RskTransactionHelper} rskTxHelper 
+ * @param {BtcTransactionHelper} btcTxHelper 
+ */
+const disableWhitelisting = async (rskTxHelper, btcTxHelper, blockDelay = 1, mineDelayBlocks = true) => {
```

Comment:
This already exists in 2wp-utils https://github.com/rsksmart/rootstock-integration-tests/blob/main/lib/2wp-utils.js#L249

Reply:
Updated!